tt_metal/jit_build/build.cpp

```diff
@@ -433,9 +433,9 @@ JitBuildIdleEthernet::JitBuildIdleEthernet(const JitBuildEnv& env, const JitBuil
                 env_.cflags_ + "-Os " + "-fno-tree-loop-distribute-patterns ";  // don't use memcpy for cpy loops
 
             this->defines_ +=
-                "-DCOMPILE_FOR_IDLE_ERISC "
+                "-DCOMPILE_FOR_IDLE_ERISC=0 "
                 "-DERISC "
-                "-DRISC_B0_HW ";
+                "-DRISC_B0_HW ";    // do we need this for BH?
```

Comment:
I think this flag was toggling the IRAM build path for WH B0, I'm almost sure that this i not needed in Metal at all.